src/aggregate/expr/parser.y

```diff
@@ -1,13 +1,13 @@
-/*
- * Copyright Redis Ltd. 2016 - present
- * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
- * the Server Side Public License v1 (SSPLv1).
- */
-
+/*
+ * Copyright Redis Ltd. 2016 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+
 
 %name RSExprParser_Parse
 
-%left AND OR NOT.
+%left OR AND NOT.
```

Comment:
Changing precedence - `AND` will have higher precedence than `OR` - isn't it breaking?

Reply:
You're right... Let's wait until 2.12/3.0 for it. Will revert.